src/sentry/integrations/msteams/unlink_identity.py

```diff
@@ -30,7 +31,13 @@ class MsTeamsUnlinkIdentityView(BaseView):
     @transaction_start("MsTeamsUnlinkIdentityView")
     @never_cache
     def handle(self, request, signed_params):
-        params = unsign(signed_params)
+        try:
+            params = unsign(signed_params)
+        except (SignatureExpired, BadSignature):
```

Comment:
Is this supposed to be a change for Slack? This is an MSTeams file.

Reply:
Sorry this got caught in a `git add .`. I'll make a separate PR.